__mocks__/handlers/pull-requests.handler.ts

```diff
@@ -0,0 +1,15 @@
+import { rest } from 'msw';
+import { mockGetOpenPrs, mockGetStalePrs } from '../db/prs';
+const URL = process.env.NEXT_PUBLIC_BASE_URL;
+
+export const openPrsHandler = [
+    rest.get(`${URL}/pullrequests/open?page=1&size=3`, (_, res, ctx) => {
+        return res(ctx.status(200), ctx.json(mockGetOpenPrs));
+    }),
+];
+
+export const stalePrsHandler = [
+    rest.get(`${URL}/pullrequests/stale?page=1&size=3`, (_, res, ctx) => {
+        return res(ctx.status(200), ctx.json(mockGetStalePrs));
+    }),
+];
```

Comment:
we don't need to make 2 separate handlers for a single API
`getPrsHandler`  and both will go in it

Reply:
Handlers are just used for tests in mocks so I think they will be helpful for people who want to understand what scenarios will this query be used for

Reply:
Since the status site has 2 pages based on PRs 1 for stale and one for open keeping 2 handlers in tests bifurcates and explicitly showing both the use case can be implemented.

Reply:
thanks for explaining it. Please re-review.